_test/test_change_crystal/test_change_crystal_1.m

```diff
@@ -9,8 +9,9 @@
 
 % -----------------------------------------------------------------------------
 % Add common functions folder to path, and get location of common data
-addpath(fullfile(fileparts(which('horace_init')),'_test','common_functions'))
-common_data_dir=fullfile(fileparts(which('horace_init')),'_test','common_data');
+horace_root = fileparts(fileparts(which('horace_init')));

```

Comment:
this single line appears dozens of times; would it be helpful to define it as a tiny helper function `get_horace_root()`?

Reply:
again, the code becomes unclear -- root -- is it the place where matlab code resides? is it where .git is located?